rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "nightly-2022-11-14"
+channel = "nightly-2022-12-15"
 components = ["rustfmt", "clippy", "rust-src"]
```

Comment:
Maybe you can help add `miri` to components. :heart:

Reply:
Be advised that rust nightly builds not always come out with `miri` available.